change/@fluentui-react-textarea-da38b5b6-139a-42de-b425-abceddc48a77.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 6 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27920&commitId=51ba94e8ec78745c3648c3cd1637cf40c2c217a4&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1930 screenshots unchanged, 0 screenshots with different dimensions and 6 screenshots with visible difference.


  <details><summary><b>unknown</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.mouseDownDecrement.chromium.png|1831817| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.mouseDownDecrement.chromium.png|1832400| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.mouseDownDecrement.chromium.png|1827811| Changed|
|SpinButton Converged.Appearance- filled-darker.mouseDownDecrement.chromium.png|1827814| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.mouseDownDecrement.chromium.png|1821419| Changed|
|SpinButton Converged.Appearance- filled-lighter.mouseDownDecrement.chromium.png|1821426| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>

Reply:
These are not related to the Textarea changes